ga4gh/cli.py

```diff
@@ -521,7 +552,7 @@ def addNameArgument(parser):
 def addClientGlobalOptions(parser):
     parser.add_argument('--verbose', '-v', action='count', default=0)
     parser.add_argument(
-        "--workarounds", "-w", default='', help="The workarounds to use")
```

Comment:
I think the old way is better actually - having a default of the empty makes creating the set of workarounds simpler and doesn't need a conditional.